### ISSUE TITLE
包含shift+command或option组合键会引起中英状态的切换

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/rimeime#341]
```
程序员工作时会频繁使用各种快捷键, 
使用command,shift,option,control都非常之高, 各位都是开发者, 
应该感触很深. 
非主动引发的输入法切换会极大的影响输入法的使用体验. 
这对一般用户影响不是特别大, 
不过我想鼠须管的用户程序员的比例还是相当之高的.

这个bug基本如标题所述, 
我在使用shift+command+左右方向键选中一行或者option选中单词时
候频繁引发中英文切换. 
我希望你们监听shift键被松开的消息时候判断先前是否按下了
其他控制键,不同的松开的顺序最好都考虑到.

严格来说这并不属于鼠须管专有的bug应该是OSX的bug,我试过的�
��入法都会如此. 但我希望你们可以在鼠须管中解决.


所得的結果是這個樣：按shift+其他键的组合键会引起中英文��
�态切换
預期結果該是那個樣：只有单独按下shift键时候才切换中英文
状态

請提供以下信息，幫我們定位問題。

1. 輸入法的款式：鼠鬚管
   輸入法版本號：0.9.14.5

2. 操作系統類型：Mac
   操作系統版本：10.8.2

3. 輸入方案名稱：明月

還想再說幾句：
请原谅我把优先级设的很高, 
因为这在我选常用输入法的考虑因素里占的优先级很高.
```

Original issue reported on code.google.com by `ikun...@gmail.com` on 14 Jan 2013 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
Issue 285 has been merged into this issue.
```

Original comment by `chen....@gmail.com` on 14 Jan 2013 at 7:03

**Comment by GoogleCodeExporter** [comment]
```
可以试试。
```

Original comment by `chen....@gmail.com` on 14 Jan 2013 at 7:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in Squirrel 0.9.15.
```

Original comment by `chen....@gmail.com` on 16 Jan 2013 at 10:07
- Changed state: **Fixed**
